command/ca/init.go

```diff
@@ -464,6 +482,34 @@ func initAction(ctx *cli.Context) (err error) {
 			dnsNames = append(dnsNames, strings.TrimSpace(name))
 		}
 
+		if useContext {
+			context := ctx.String("context")
+			if context == "" {
+				context = dnsNames[0]
+			}
+			authority := ctx.String("authority")
+			if authority == "" {
+				authority = dnsNames[0]
+			}
+			profile := ctx.String("profile")
+			if profile == "" {
+				profile = dnsNames[0]
+			}
```

Comment:
What would you propose doing if it already exists? PromptYesNo for continue?

Reply:
If it's going to start overwriting it will ask the user about it. Not sure we need to defend against that. They can always say no and change the names.

Reply:
ok we can always change it later if needed.